core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -143,4 +144,23 @@ package object config {
       .internal()
       .stringConf
       .createWithDefaultString("AES/CTR/NoPadding")
+
+  private[spark] val DRIVER_BIND_ADDRESS = ConfigBuilder("spark.driver.bindAddress")
+    .doc("Address where to bind network listen sockets on the driver.")
+    .stringConf
+    .createWithDefault(Utils.localHostName())
```

Comment:
This is a broken change. If a user uses `spark.driver.host` to specify the bind address, it won't work now. Right?